### PR TITLE
NPM debug API's and pprof

### PR DESCRIPTION
**Pull request by matmerr**
**Reason for Change**:

- Start adding NPM debug API's and integrate client into ACN CLI. Actual objects TBD but starting initiative: 

```
./acn npm get npmgr
{
        "NsMap": {
                "all-namespaces": {
                        "PodMap": {}
                },
                "ns-default": {
                        "PodMap": {}
                },
                "ns-kube-node-lease": {
                        "PodMap": {}
                },
                "ns-kube-public": {
                        "PodMap": {}
                },
                "ns-kube-system": {
                        "PodMap": {}
                },
                "ns-perf": {
                        "PodMap": {}
                },
                "ns-testa": {
                        "PodMap": {}
                },
                "ns-testb": {
                        "PodMap": {}
                }
        },
        "PodMap": {
                "13914f4a-19a0-43c4-b1b9-5c0ce53280e1": "10.240.5.188",
                "34bac546-a41f-4816-a524-6c99876b0138": "10.240.2.165",
                "399f16d3-fed7-4b8d-8eb6-3e2ba6e51e9b": "10.240.3.251",
                "5a22701c-2726-4d96-8122-cbba1c07d348": "10.240.5.41",
                "5ed5293d-24b3-4989-9928-b0b42a53e1a6": "10.240.2.49",
                "618b6b68-09e3-4e86-b6c8-2e39fe64b645": "10.240.5.64",
                "7ca112b5-173f-4793-bbbe-7194f775853f": "10.240.3.112",
                "89b17391-97ac-4dc2-93b2-83a4e00e1a27": "10.240.7.50",
                "bd87f2ff-f833-4dcb-a8c6-a00ea13c41ac": "10.240.1.37",
                "c3baee43-05db-4957-8771-ec062185af4b": "10.240.6.55",
                "c4554edb-ac9c-4ad6-9cdf-7973f0b39d7c": "10.240.1.12",
                "c91acd83-0f63-4820-9016-ae19f4a88581": "10.240.2.246",
                "d88baeb7-9a3a-42d9-b5db-4b026ece8546": "10.240.5.61",
                "e4a1f688-a6cd-4019-aa82-08ba0f34a879": "10.240.1.209"
        },
        "TelemetryEnabled": true
}
```
- Add pprof listening (TODO: enable/disable this through config)
- Add basic test to Prometheus HTTP endpoint

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Comment by vakalapa** [comment]
Can you rebase this PR to the latest changes,

File: npm/http/api/api.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	NodeMetricsPath    = "/node-metrics"
+	ClusterMetricsPath = "/cluster-metrics"
+	NPMMgrPath         = "/npm/v1/debug/manager"
+)
+
+type DescribeIPSetRequest struct {
+	ipsetname string `json:"name"`
+}
+
+type DescribeIPSetResponse struct {
```
Comment (matmerr) [inline npm/http/api/api.go:15]: to be used eventually

File: npm/npm.go [hunk 2]
```diff
@@ -165,6 +165,7 @@ func (npMgr *NetworkPolicyManager) backup() {
 
 // Start starts shared informers and waits for the shared informer cache to sync.
 func (npMgr *NetworkPolicyManager) Start(stopCh <-chan struct{}) error {
+
```
Comment (matmerr) [inline npm/npm.go:168]: remove

File: npm/npm.go [hunk 3]
```diff
@@ -118,11 +118,11 @@ func (npMgr *NetworkPolicyManager) SendClusterMetrics() {
 	for {
 		<-heartbeat
 		npMgr.Lock()
-		podCount.Value = float64(len(npMgr.podMap))
+		podCount.Value = float64(len(npMgr.PodMap))
```
Comment (matmerr) [inline npm/npm.go:121]: make sure this is up to date with master

**Pull request closed without merging**